<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompetitionManagerWindow</class>
 <widget class="QDialog" name="CompetitionManagerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1284</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Przebieg konkursu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(236, 236, 236);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>271</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_startList">
    <item>
     <widget class="QLabel" name="label_title">
      <property name="font">
       <font>
        <family>Quicksand Medium</family>
        <pointsize>18</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color: rgb(26, 95, 180);
}</string>
      </property>
      <property name="text">
       <string>Lista startowa</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListView" name="listView_startList">
      <property name="styleSheet">
       <string notr="true">QListView{
	background-color: rgb(255, 255, 255);
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>634</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_nextJumper">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Quicksand [PfEd]</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: black;</string>
        </property>
        <property name="text">
         <string>NastÄ™pny zawodnik: </string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_nameAndSurname">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Quicksand [PfEd]</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(48, 48, 48);</string>
          </property>
          <property name="text">
           <string>Name Surname</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_flag">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_jump">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Quicksand Light [PfEd]</family>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: 1px solid rgb(0, 59, 23);
border-radius: 6px;
color: rgb(0, 0, 0);
background-color: rgb(138, 200, 255);
}
QPushButton:hover{
background-color: rgb(158, 220, 255);
}</string>
        </property>
        <property name="text">
         <string>Skok</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_manipulateJump">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Quicksand [PfEd]</family>
          <pointsize>11</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: 1px solid rgb(0, 59, 23);
border-radius: 6px;
color: rgb(0, 0, 0);
background-color: rgb(253, 202, 96);
}
QPushButton:hover{
background-color: rgb(255, 222, 116);
}</string>
        </property>
        <property name="text">
         <string>Manipuluj skokiem</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tableView_results">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>811</width>
     <height>531</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView{
	background-color: white;
}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
